### PR TITLE
[SASS-1228 Confirmation page crystallisation view updates]

### PR DESCRIPTION
**Pull request by juamalBlackman**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-1228](https://jira.tools.tax.service.gov.uk/browse/SASS-1228)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of master?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current master that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of master?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/views/TaxReturnReceivedView.scala.html [hunk 1]
```diff
@@ -61,20 +60,12 @@
...
-                ),
-                value = Value(
-                    classes = "govuk-!-width-one-quarter",
-                    content = HtmlContent(bigDecimalCurrency(summaryData.incomeTaxAndNationalInsuranceContributions.toString))
-                )
             )
         )
     ))
 
+    <br>
```
Comment (dnyuar) [inline app/views/TaxReturnReceivedView.scala.html:67]: I don't think using <br> tags to create spacing between paragraphs is the best way to go about it. Can cause problems for people using screen readers

**Review by dnyuar — CHANGES_REQUESTED** [review]
A couple dependencies can be updated
Screenshot 2021-09-21 at 10 12 07

File: app/views/TaxReturnReceivedView.scala.html (hunk 1 above)
Comment (dnyuar) [reply app/views/TaxReturnReceivedView.scala.html:67]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Comment (juamalBlackman) [reply app/views/TaxReturnReceivedView.scala.html]: fixed All

**Review by dnyuar — APPROVED** [review]

**Pull request merged**